javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,25 @@
+function getFullname(firstname, lastname, useFormalName, sex) {
+  if (firstname === "" || lastname === "") {
+    alert("Please enter a full name!"); //a bit of validation, so the data is provided
+  } else if (useFormalName === true) {
```

Comment:
The alert can also just be a message  - i get an error with the current implementation. 
so e.g. return "Please enter a full name"

Reply:
I will fix it :) Thank you!